setup.py

```diff
@@ -239,7 +239,7 @@ def write_version(filename=os.path.join(*['airflow',
 vertica = ['vertica-python>=0.5.1']
 webhdfs = ['hdfs[dataframe,avro,kerberos]>=2.0.4']
 winrm = ['pywinrm==0.2.2']
-zendesk = ['zdesk']
+zendesk = ['zdesk>=2.7.0']
```

Comment:
Why do we want to pin the version? The change to zendesk_hook.py  makes it compatible with all versions doesn't it?

Reply:
checked previous zdesk releases (down to version 2.0.2), you are correct